widget/src/main/java/com/ticketmaster/servos/widget/FlatButton.java

```diff
@@ -0,0 +1,38 @@
+package com.ticketmaster.servos.widget;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.drawable.Drawable;
+import android.graphics.drawable.LayerDrawable;
+import android.support.v7.widget.AppCompatButton;
+import android.util.AttributeSet;
+
+public class FlatButton extends AppCompatButton {
```

Comment:
Could you Javadoc the class with just a brief explanation of why it exists and how to use it's special features?